src/Symfony/Component/Yaml/Parser.php

```diff
@@ -733,10 +742,12 @@ private function parseValue(string $value, int $flags, string $context)
         }
 
         try {
-            if ('' !== $value && '{' === $value[0]) {
-                return Inline::parse($this->lexInlineMapping($value), $flags, $this->refs);
-            } elseif ('' !== $value && '[' === $value[0]) {
-                return Inline::parse($this->lexInlineSequence($value), $flags, $this->refs);
+            if ('' !== $value) {
+                if ('{' === $value[0]) {
```

Comment:
Does this change really make a significant difference?

Reply:
Logically it means fewer checks because `'' !== $value` is only done once instead of twice - but I did not measure this specifically.

Reply:
reverted